bin/analyze_hoc_activity

```diff
@@ -101,15 +101,21 @@ def main()
     # Since s3 (which the binary relies on) may fail, we retry a few times
     # hoping for success.
     3.times do
-      # Note that by executing via backticks, the global variable $? is set.
+      # Note that by executing via backticks, the global variable $? is set to a Process::Status object.
       lines_of_code_from_milestone = `#{deploy_dir('bin','count-lines-of-code-from-milestone-logs_v2')}`.to_i
```

Comment:
have you looked at `count-lines-of-code-from-milestone-logs_v2` to see if it would benefit from better logging as well?